indico/modules/events/registration/controllers/display.py

```diff
@@ -90,10 +91,12 @@ def _process(self):
         else:
             criteria = RegistrationForm.is_scheduled
         regforms = regforms.filter(criteria).order_by(db.func.lower(RegistrationForm.title)).all()
+
         if len(regforms) == 1:
             return redirect(url_for('.display_regform', regforms[0]))
-        return self.view_class.render_template('display/regform_list.html', self._conf, event=self.event_new,
-                                               regforms=regforms)
+        return self.view_class.render_template('display/regform_list.html', self._conf,
+                                               event=self.event_new, regforms=regforms,
+                                               registration_counts=get_registration_counts_dict(self.event_new, True))
```

Comment:
I would use `publish=True` here; it's usually clearer for optional boolean args. Since it probably won't fit in the line that way, I'd simply assign it to a variable before and use that one here.